acquire-video-runtime/src/runtime/channel.c

```diff
@@ -165,28 +165,28 @@ channel_read_map(struct channel* self, struct channel_reader* reader)
     reader_initialize(self, reader);
 
     size_t* const cycle = self->holds.cycles + reader->id - 1;
-    size_t* const pos = self->holds.pos + reader->id - 1;
-    uint8_t* out = self->data + *pos;
+    size_t* const tail = self->holds.pos + reader->id - 1;
+    uint8_t* out = self->data + *tail;
```

Comment:
this is a nit and I think you can judge best but I still like `pos`. It's probably worth noting in a comment that channel_reader holds the next reader tail.